google/cloud/storage/_helpers.py

```diff
@@ -190,7 +190,9 @@ def reload(
                                             blob's current metageneration does not match the given value.
 
         :type retry: google.api_core.retry.Retry
-        :param retry: (Optional) How to retry the RPC.
+        :param retry: (Optional) How to retry the RPC. To modify the default retry behavior,
+            create a new retry object modeled after this one by calling it a ``with_XXX`` method.
+            See: https://googleapis.dev/python/google-api-core/latest/retry.html for details.
```

Comment:
(Optional) How to retry the RPC. A `None` value will disable retries. A `google.api_core.retry.Retry` value will enable retries, and the object will define retriable response codes and errors and configure backoff and timeout options.

A `google.cloud.storage.retry.ConditionalRetryPolicy` value wraps a Retry object and activates it only if certain conditions are met. This class exists to provide safe defaults for RPC calls that are not technically safe to retry normally (due to potential data duplication or other side-effects) but become safe to retry if a condition such as `if_metageneration_match` is set.

See the google.cloud.storage.retry source code and docstrings for information on retry types and how to configure them.